android/src/main/java/expo/modules/xmtpreactnativesdk/wrappers/GroupWrapper.kt

```diff
@@ -18,6 +18,10 @@ class GroupWrapper {
                 "topic" to group.topic,
                 "creatorInboxId" to group.creatorInboxId(),
                 "isActive" to group.isActive(),
+                "name" to group.name,
+                "imageUrlSquare" to group.imageUrlSquare,
+                "description" to group.description,
+                "pinnedFrameUrl" to group.pinnedFrameUrl
```

Comment:
Probably no cost in adding these as well here. But I don't really see a scenario where you would need them in a group list context?